frontend/src/hooks/apiHooks.ts

```diff
@@ -57,12 +58,14 @@ export const useApiPost = <T>(apiPath: string) => {
 					'Error fetching data from server with status code: ' + response.status
 				);
 			}
+
+			setSuccess(true);
```

Comment:
What's the point of having success when error is thrown every time it's not a success?
Likewise, success is going to be true every time after the first, whether it's a success or not, right?

Reply:
Well, it's mostly there as an alternative if you don't actually care about the error, and only want the actual loading state and whether it succeded or not in the UI. Kind of like a shorthand. It's supposed to reset when starting the function again, I think I just forgot...